learning-development/r.qmd

```diff
@@ -564,6 +564,10 @@ mutated <- mutate(mtcars, new_var = wt ** 2)
 
 ---
 
+## Troubleshooting
+
+---
```

Comment:
Don't need a line break underneath a level 2 header

Reply:
It automatically adds one in in the quarto styling so looks like this currently, just need to delete it and it should look fine 
![Uploading image.png…]()

Reply:
Ugh I do this by accident EVERY TIME! Sorry!